config.yml

```diff
@@ -6,47 +6,39 @@ CI_HELPER_VERSION: &CI_HELPER_VERSION 0.0.6
 ci_helper_test: &ci_helper_test ci-helper version -e
 MODD_VERSION: &MODD_VERSION 0.5
 
+ansible_tests: &ansible_tests
+    cmd:
+      - python --version
+      - pip --version
+      - ansible --version
+      - ansible-playbook --version
+
 ansible:
-  "2.5.5":
+  "2.8":
+    template_vars:
+      VERSION: 3.8
     build_args:
-      CI_HELPER_VERSION: *CI_HELPER_VERSION
-      VERSION: 2.5.5
-    test_config:
-      cmd:
-        - *ci_helper_test
-        - ansible --version
-        - ansible-playbook --version
-        - python --version
+      ANSIBLE_VERSION: 2.8.8
+    test_config: *ansible_tests
 
 arachni:
   "1.5":
     build_args:
       ARACHNI_VERSION: 1.5.1
       ARACHNI_WEB_UI_VERSION: 0.5.12
-      CI_HELPER_VERSION: *CI_HELPER_VERSION
     test_config:
       cmd:
         - arachni --version
-        - *ci_helper_test
 
 aws:
   "1":
     test_config:
       cmd:
         - aws --version
-        - node --version
-        - npm --version
         - pip --version
-        - pip list | grep boto3
-        - pip list | grep pyyaml
         - pipenv --version
-        - *ci_helper_test
-    build_args:
-      CI_HELPER_VERSION: *CI_HELPER_VERSION
-      NODE_VERSION: 10.16.3-r0
-      PIP_VERSION: 18.1
-      PIPENV_VERSION: 2018.10.13
-      PY_YAML_VERSION: 3.13
+    template_vars:
+      VERSION: 3.8
```

Comment:
I know this is the base python image's version, but not everybody will. We should name it either `PYTHON_VERSION` to be explicit, or `BASE_IMAGE_VERSION` to be more generic (or something like that).

Reply:
ie: `VERSION` was ok for our java image, since it's java's version. Now it's a completly different thing.

Reply:
Or just read the README about the variable role. Maybe using `BASE_IMAGE_VERSION` should be better tho

Reply:
Switched to BASE_IMAGE_VERSION for every image. More meaningfull, and README updated.